### PR TITLE
Updates csproj release notes to point to Github release notes

### LINKED ISSUE #923: Update Release Notes in csproj file to point to release notes on GitHub
Rather than duplicating release notes in the csproj and on GitHub, we should add a link to the GitHub Releases page for this repo into the csproj release notes.  I recommend just pointing to e.g https://github.com/microsoft/OpenAPI.NET/releases/tag/1.3.3  We can update the version each time we update the library version in the csproj.

### PR DESCRIPTION
**Pull request by MaggieKimani1**
Fixes #923

### CODE REVIEW

**Review by baywet — APPROVED** [review]

**Review by baywet — CHANGES_REQUESTED** [review]
actually, the same change should probably be done for openapi.net and readers.

**Comment by MaggieKimani1** [comment]
> actually, the same change should probably be done for openapi.net and readers.

Oh I missed that. Resolving..

File: src/Microsoft.OpenApi/Microsoft.OpenApi.csproj [hunk 1]
```diff
@@ -16,9 +16,7 @@
         <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
         <PackageTags>OpenAPI .NET</PackageTags>
         <RepositoryUrl>https://github.com/Microsoft/OpenAPI.NET</RepositoryUrl>
-        <PackageReleaseNotes>
-- Adds support for c-style hex notation strings. #908
-        </PackageReleaseNotes>
+        <PackageReleaseNotes>https://github.com/microsoft/OpenAPI.NET/releases/tag/1.3.3</PackageReleaseNotes>
```
Comment (baywet) [inline src/Microsoft.OpenApi/Microsoft.OpenApi.csproj:19]:
```suggestion
        <PackageReleaseNotes>https://github.com/microsoft/OpenAPI.NET/releases</PackageReleaseNotes>
```
```suggestion
        <PackageReleaseNotes>https://github.com/microsoft/OpenAPI.NET/releases/tag/1.3.3</PackageReleaseNotes>
```

File: src/Microsoft.OpenApi.Readers/Microsoft.OpenApi.Readers.csproj [hunk 2]
```diff
@@ -15,9 +15,7 @@
         <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
         <PackageTags>OpenAPI .NET</PackageTags>
         <RepositoryUrl>https://github.com/Microsoft/OpenAPI.NET</RepositoryUrl>
-        <PackageReleaseNotes>
-- Fixed a bug where contact information would not read properly. #892
-        </PackageReleaseNotes>
+        <PackageReleaseNotes>https://github.com/microsoft/OpenAPI.NET/releases/tag/1.3.3</PackageReleaseNotes>
```
Comment (baywet) [inline src/Microsoft.OpenApi.Readers/Microsoft.OpenApi.Readers.csproj:18]:
```suggestion
        <PackageReleaseNotes>https://github.com/microsoft/OpenAPI.NET/releases</PackageReleaseNotes>
```
```suggestion
        <PackageReleaseNotes>https://github.com/microsoft/OpenAPI.NET/releases/tag/1.3.3</PackageReleaseNotes>
```

**Review by baywet — APPROVED** [review]

**Pull request merged**